Adicionando 1847 e 1848 (#54)
petbccufscar/URI: Iniciante/1847/README.md

* Se a temperatura subiu do 1º para o 2º dia e do 2º para o 3º, mas subiu do 2º para o 3º no mínimo o tanto que subira do 1º para o 2º, as pessoas ficam felizes (quarta figura).
* Se a temperatura desceu do 1º para o 2º dia e do 2º para o 3º, mas desceu do 2º para o 3º menos do que descera do 1º para o 2º, as pessoas ficam felizes (quinta figura).
* Se a temperatura desceu do 1º para o 2º dia e do 2º para o 3º, mas desceu do 2º para o 3º no mínimo o tanto que descera do 1º para o 2º, as pessoas ficam tristes (sexta figura).
* Se a temperatura permaneceu constante do 1º para o 2º dia, as pessoas ficam felizes se subiu do 2º para o 3º dia ou tristes caso contrário (respectivamente, sétima e oitava figuras).

![Graficos](https://resources.urionlinejudge.com.br/gallery/images/problems/UOJ_1847.jpg)
###### Problema completo: https://www.urionlinejudge.com.br/judge/pt/problems/view/1847

# Resolução:
Para resolver esse problema receberemos as temperaturas registradas no 1º, 2º e 3º dia, respectivamente, e ao compara-las exibiremos o humor do povo de westeros de acordo com as tendências climáticas, utilizando as figuras apresentadas no problema como referência.

> Para começar declaramos as váriaveis dos 3 dias que receberão valores referentes ao clima:

Reviewer: Mudar a posição do acento de variáveis.

> Para começar declaramos as váriaveis dos 3 dias que receberão valores referentes ao clima:

Reviewer 2: ```suggestion
Para começar declaramos as variáveis dos 3 dias que receberão valores referentes ao clima:
```

```c
int a, b, c;
```

Depois recebemos as temperaturas com o comando `scanf`:
```c
scanf("%d %d %d", &a, &b, &c);
```

> Como existem varias combinações semelhantes que influenciam de forma diferente o humor, primeiro iremos comparar se o clima do primerio dia é mais quente que do segundo. Desse forma, agrupamos 3 possibilidades(figuras: 1, 5 e 6). 

Reviewer: Colocar acento em várias.

> Como existem varias combinações semelhantes que influenciam de forma diferente o humor, primeiro iremos comparar se o clima do primerio dia é mais quente que do segundo. Desse forma, agrupamos 3 possibilidades(figuras: 1, 5 e 6). 

Reviewer 2: ```suggestion
Como existem várias combinações semelhantes que influenciam de forma diferente o humor, primeiro iremos comparar se o clima do primeiro dia é mais quente que do segundo. Desse forma, agrupamos 3 possibilidades(figuras: 1, 5 e 6). 
```

* Se, sim imprimimos o humor triste: :(
* se não imprimimos o humor feliz: :)
```c
if(c<b && (b-c)>=(a-b))
		printf(":(\n");
	else 
	    printf(":)\n");
```

Depois com o comando`else if` comparamos se a temperatura do primeiro dia é menor do que a do segundo. Assim, agrupamos outras 3 possibilidades(figuras: 2, 3 e 4)
```c
else if(b>a){
	/*
	Próximo Bloco de Código
	*/
}
```
Observando as possibilidades dessas 3 figuras percebemos duas represetam o humor triste(figuras 2 e 3) e uma que representa o humor feliz(figura 4). Para separa-las verificamos se a temperatura do 3º dia é *maior* que a do 2º e se a diferença entre as temperaturas do 2º e 3º dia é *maior ou igual* a diferença das temperaturas do 1º e 2º dia.  
* Se, sim imprimimos o humor feliz: :)
* se não imprimimos o humor triste: :(

```c
if(c>b && (c-b)>=(b-a))
		printf(":)\n");
	else
		printf(":(\n");
```

> E por ultimo verificamos se a temperatura do primeiro dia é igual a do segundo.  Assim, agrupamos as ultimas 2 possibilidades(figuras: 7 e 8) e verificamos se a temperatura do 3º dia é *maior* que a do 2º.

Reviewer: Colocar acento em último e última

> E por ultimo verificamos se a temperatura do primeiro dia é igual a do segundo.  Assim, agrupamos as ultimas 2 possibilidades(figuras: 7 e 8) e verificamos se a temperatura do 3º dia é *maior* que a do 2º.

Reviewer 2: ```suggestion
E por último verificamos se a temperatura do primeiro dia é igual a do segundo.  Assim, agrupamos as últimas 2 possibilidades(figuras: 7 e 8) e verificamos se a temperatura do 3º dia é *maior* que a do 2º.
```

```c
else if(b==a && c>b)
		printf(":)\n");
	else
		printf(":(\n");
```
* Se, sim imprimimos o humor feliz: :)
* Se não imprimimos o humor triste: :(

> ##### Para aprender um pouco mais sobre a funçao `fgets`:[fgets](https://aprendendoc.wordpress.com/2012/02/03/entrada-e-saida-de-dados/)

Reviewer: Nessa parte poderia ser sobre condicionais e operadores relacionais ou lógicos.
